docs/stencil-docs/customizing-checkout/optimized-one-page-checkout.md

```diff
@@ -63,17 +63,17 @@ Below are the classes provided in Cornerstone's <span class="fn">optimized-check
 | `.optimizedCheckout-form-label` | Styles form fields like `Email Address` – the field's text-label color. |
 | `.optimizedCheckout-form-input` | Styles form fields like `Email Address` – the entry box's background and border colors. |
 
-## Providing Customizable Options in Store Design
+## Providing customizable options in Store Design
 
-You can determine which aspects of Optimized One-Page Checkout merchants will be able to customize with the Store Design tool.
+You can determine which aspects of Optimized One-Page Checkout merchants will customize with the Store Design tool.
 
 In Cornerstone's <span class="fn">optimized-checkout.scss</span>, each SCSS class and property available has a corresponding key/value pair in the <span class="fn">config.json</span> file, which can be used as the `id` value in schema.json when [creating customizable Store Design options](/stencil-docs/store-design/schema-json-metadata).

```

Comment:
Updated the link since /stencil-docs/store-design/schema-json-metadata redirects to https://github.com/bigcommerce/cornerstone/blob/master/schema.json
```suggestion
In Cornerstone's <span class="fn">optimized-checkout.scss</span>, each SCSS class and property available has a corresponding key/value pair in the <span class="fn">config.json</span> file, which can be used as the `id` value in schema.json when [creating customizable Page Builder options](https://github.com/bigcommerce/cornerstone/blob/master/schema.json).

```